src/services/registrate.js

```diff
@@ -0,0 +1,20 @@
+const ENDPOINT = `${import.meta.env.VITE_SERVER_URL}/api/v1/users`
+
+export default function registrate ({ email, password, password_confirmation }) {
+  return (
+    fetch(`${ENDPOINT}`, {
```

Comment:
[axios ](https://www.npmjs.com/package/axios)could help you save a lot of time and boilerplate code when making calls to api's. this is just a suggestion,